### ISSUE TITLE
mb-melgan training from checkpoint shoots up loss in between 

### ISSUE THREAD

**Issue by manmay-nakhashi** [TensorSpeech/TensorFlowTTS#186]
[train]:   0%|                                                                                                                                                   | 1200/4000000 
[WARNING] (Step: 1200) train_adversarial_loss = 0.7722.
[WARNING] (Step: 1200) train_subband_spectral_convergence_loss = 1.3400.
[WARNING] (Step: 1200) train_subband_log_magnitude_loss = 0.7954.
[WARNING] (Step: 1200) train_fullband_spectral_convergence_loss = 1.1050.
[WARNING] (Step: 1200) train_fullband_log_magnitude_loss = 0.9838.
[WARNING] (Step: 1200) train_gen_loss = 4.0427.
[WARNING] (Step: 1200) train_real_loss = 0.1281.
[WARNING] (Step: 1200) train_fake_loss = 0.0849.
[WARNING] (Step: 1200) train_dis_loss = 0.2130.
[train]:   0%|                                                                                                                                                   | 1400/4000000 [11:45<538:42:22,  2.06it/s][WARNING] (Step: 1400) train_adversarial_loss = 273810.7500.
[WARNING] (Step: 1400) train_subband_spectral_convergence_loss = 222.8679.
[WARNING] (Step: 1400) train_subband_log_magnitude_loss = 1.4122.
[WARNING] (Step: 1400) train_fullband_spectral_convergence_loss = 83.9123.
[WARNING] (Step: 1400) train_fullband_log_magnitude_loss = 1.6522.
[WARNING] (Step: 1400) train_gen_loss = 684681.7500.
[WARNING] (Step: 1400) train_real_loss = 3.8607.
[WARNING] (Step: 1400) train_fake_loss = 380371.0938.
[WARNING] (Step: 1400) train_dis_loss = 380374.9062.

**Comment by dathudeptrai** [comment]
@manmay-nakhashi seems mb-melgan have a some data-related problems. Can you try to apply tanh function after pqmf.synthesis first ?

**Comment by manmay-nakhashi** [comment]
@dathudeptrai sure

**Comment by manmay-nakhashi** [comment]
@dathudeptrai it seems training is stabilized for now , i'll still monitor it for a while and get back to you. :))

**Comment by dathudeptrai** [comment]
@tekinek, @ZDisket. Seem tanh is the solution :)).

**Comment by manmay-nakhashi** [comment]
@dathudeptrai 
[WARNING] (Step: 5600) train_adversarial_loss = 0.6759.
[WARNING] (Step: 5600) train_subband_spectral_convergence_loss = 1.1389.
[WARNING] (Step: 5600) train_subband_log_magnitude_loss = 0.7037.
[WARNING] (Step: 5600) train_fullband_spectral_convergence_loss = 0.8178.
[WARNING] (Step: 5600) train_fullband_log_magnitude_loss = 0.8724.
[WARNING] (Step: 5600) train_gen_loss = 3.4561.
[WARNING] (Step: 5600) train_real_loss = 0.1297.
[WARNING] (Step: 5600) train_fake_loss = 0.1290.
[WARNING] (Step: 5600) train_dis_loss = 0.2587.
[train]:   0%|▏                                                                                                                                                  | 5800/4000000 [47:40<538:58:55,  2.06it/s][WARNING] (Step: 5800) train_adversarial_loss = 83333.0234.
[WARNING] (Step: 5800) train_subband_spectral_convergence_loss = 791.8905.
[WARNING] (Step: 5800) train_subband_log_magnitude_loss = 1.4209.
[WARNING] (Step: 5800) train_fullband_spectral_convergence_loss = 219.3141.
[WARNING] (Step: 5800) train_fullband_log_magnitude_loss = 1.6943.
[WARNING] (Step: 5800) train_gen_loss = 208839.6562.
[WARNING] (Step: 5800) train_real_loss = 12.5621.
[WARNING] (Step: 5800) train_fake_loss = 180993.9062.
[WARNING] (Step: 5800) train_dis_loss = 181006.4375.
happened again !!!!

**Comment by dathudeptrai** [comment]
@manmay-nakhashi can you follow the solution here #125. Mask the loss >= value :)).

**Comment by manmay-nakhashi** [comment]
@dathudeptrai i have started training from english checkpoint , do i still need to train generator separately ?

**Comment by dathudeptrai** [comment]
@manmay-nakhashi even you load english checkpoint and continue to training, you still need follow the default config :)).

**Comment by manmay-nakhashi** [comment]
@dathudeptrai ok , great , i'll do that :)

**Comment by manmay-nakhashi** [comment]
@dathudeptrai solved this issue from #125.

**Comment by dathudeptrai** [comment]
@manmay-nakhashi can you share ur tensorboard, audio samples or the detail what you change to make ur training progress work ?
